lib/galaxy/workflow/modules.py

```diff
@@ -910,7 +926,9 @@ def get_inputs(self):
             optional_cond.cases = optional_cases
 
             if param_type == "text":
-                restrict_how_source = dict(name="how", label="Restrict Text Values?", type="select")
+                restrict_how_source: Union[
+                    Dict[str, Union[str, List[Dict[str, Union[str, bool]]]]]
+                ] = dict(name="how", label="Restrict Text Values?", type="select")
```

Comment:
@mr-c  I've got Pylance in my editor complaining about this line:
```
Union requires two or more type arguments
```

Can I replace this with:
```python
                restrict_how_source: Dict[
                    str, Union[str, List[Dict[str, Union[str, bool]]]]
                ] = dict(name="how", label="Restrict Text Values?", type="select")
```

Reply:
Fixed in https://github.com/galaxyproject/galaxy/pull/12899/commits/a467a7654e21663e7d7333891470c03d966a8db8